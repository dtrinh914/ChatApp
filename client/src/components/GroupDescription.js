import React from 'react';

export default function GroupDescription({description}) {
    return (
        <div>
            <p data-testid='group-description'>{description ? description : '[No description has been written]'}</p>
        </div>
    )
}
