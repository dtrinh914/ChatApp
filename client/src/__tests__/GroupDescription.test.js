import React from 'react';
import GroupDescription from '../components/GroupDescription';
import {cleanup, render} from '@testing-library/react';

afterEach(cleanup);

test('should match snapshot', ()=>{
    const {container} = render(<GroupDescription />)
    expect(container).toMatchSnapshot();
});

test('should match second snapshot', ()=>{
    const {container} = render(<GroupDescription description='This group is used for testing.' />)
    expect(container).toMatchSnapshot();
});