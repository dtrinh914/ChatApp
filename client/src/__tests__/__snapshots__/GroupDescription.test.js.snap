// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match second snapshot 1`] = `
<div>
  <div>
    <p
      data-testid="group-description"
    >
      This group is used for testing.
    </p>
  </div>
</div>
`;

exports[`should match snapshot 1`] = `
<div>
  <div>
    <p
      data-testid="group-description"
    >
      [No description has been written]
    </p>
  </div>
</div>
`;
